example/app.js

```diff
@@ -2,16 +2,25 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 const cors = require('cors')
+const compression = require('compression')
 const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware')
 const app = express()
 
+app.set('view engine', 'pug')
+app.use(compression())
 app.use(cors())
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: true }))
 app.use(awsServerlessExpressMiddleware.eventContext())
 
 app.get('/', (req, res) => {
-    res.sendFile(`${__dirname}/index.html`)
+    res.render('index', {
```

Comment:
I added support for pug templates and updated the index.html at the same time as adding gzip support. This change got swept up in the commit. See #52